app/api/events.py

```diff
@@ -858,7 +858,7 @@ def query(self, view_kwargs):
                         Event.event_type_id != None,
                         Event.event_topic_id != None,
                         Event.event_sub_topic_id != None,
-                        Event.tickets.any(and_(Ticket.deleted_at == None, Ticket.is_hidden == False, Ticket.sales_ends_at > current_time)),
+                        Event.tickets.any(and_(Ticket.deleted_at == None, Ticket.is_hidden == False, Ticket.sales_ends_at > current_time, Ticket.is_available == True)),
```

Comment:
is_available is not a column. It's a computed property. It doesn't work like this

Reply:
the query that is written in `is_available` property is that right for the condition?can that query be written here again for this work?

Reply:
Yes

Reply:
But please run and confirm before committing. Just visit /v1/events/upcoming to verify